lib/fuel/engine.rb

```diff
@@ -13,8 +13,10 @@ class Engine < ::Rails::Engine
       end
     end
 
-    initializer :copilot do
-      ActiveAdmin.application.load_paths += Dir[File.dirname(__FILE__) + '/admin']
+    if defined?(ActiveAdmin)
+      initializer :fuel do
+        ActiveAdmin.application.load_paths += Dir[File.dirname(__FILE__) + '/admin']
```

Comment:
Line is too long. [84/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.